service/src/main/java/bio/terra/catalog/service/dataset/DatasetDao.java

```diff
@@ -132,4 +132,9 @@ public List<Dataset> find(StorageSystem storageSystem, Collection<String> ids) {
     var params = Map.of("storageSystem", String.valueOf(storageSystem), "ids", ids);
     return jdbcTemplate.query(sql, params, new DatasetMapper());
   }
+
+  @ReadTransaction
+  public List<Dataset> listAllDatasets() {
+    return jdbcTemplate.query("SELECT * FROM dataset", new DatasetMapper());
```

Comment:
```suggestion
    String sql = "SELECT * FROM dataset";
    return jdbcTemplate.query(sql, new DatasetMapper());
```

Reply:
Because it doesn't have params that need mapping, I didn't see a need to abstract it out to an intermediate variable. I don't object to it, it just seems unnecessary in this case. (All the other methods do have variables passed in)